node/Cargo.toml

```diff
@@ -72,9 +72,11 @@ wasm-bindgen = { version = "0.2.57", optional = true }
 wasm-bindgen-futures = { version = "0.4.7", optional = true }
 browser-utils = { package = 'substrate-browser-utils', git = 'https://github.com/paritytech/substrate.git', rev = '00768a1f21a579c478fe5d4f51e1fa71f7db9fd4', optional = true}
 
```

Comment:
Do you mind moving these dependencies to the 'third-party' category?

Reply:
done in 6e3ebf6d2b3f8716c46acb09888a732c4adbae61